onnxruntime/python/tools/quantization/E2E_example_model/e2e_tensorrt_resnet_example.py

```diff
@@ -0,0 +1,132 @@
+import os
+import onnx
+import glob
+import scipy.io
+import numpy as np
+from onnxruntime.quantization import get_calibrator, TRTResNet50DataReader, TRTResNet50Evaluator, generate_calibration_table, write_calibration_table
+
+def get_prediction_evaluation(model_path, input_name, batch_size, dataset_path, dataset_offset, dataset_size, synset_id, providers=["TensorrtExecutionProvider"]):
+    data_reader = TRTResNet50DataReader(dataset_path, start_index=dataset_offset, end_index=dataset_offset+dataset_size, stride=dataset_size, batch_size=batch_size, model_path=model_path, input_name=input_name)
+    evaluator = TRTResNet50Evaluator(model_path, synset_id, data_reader, providers=providers)
+    evaluator.predict()
+    result = evaluator.get_result()
+    evaluator.evaluate(result)
+
+def get_calibration_table(model_path, input_name, batch_size, augmented_model_path, dataset_path, calibration_dataset_size, use_existing_augmented_model=False, use_existing_calibration_table=False):
+    calibrator = get_calibrator(model_path, None, augmented_model_path=augmented_model_path)    
+    start_index = 0
+    stride=calibration_dataset_size
+    for i in range(0, calibration_dataset_size, stride):
+        data_reader = TRTResNet50DataReader(dataset_path,start_index=start_index, end_index=start_index+stride, stride=stride, batch_size=batch_size, model_path=augmented_model_path, input_name=input_name)
+        calibrator.set_data_reader(data_reader)
+        generate_calibration_table(calibrator, model_path, augmented_model_path, False, data_reader)
+        start_index += stride
+    write_calibration_table(calibrator.get_calibration_cache())
+
+def get_synset_id(dataset_path, offset, dataset_size):
```

Comment:
this feels like it could be pulled out into some helper module that deals with imagenet dataset.
so that it's not useful for only this one specific example.